src-java/flowhs-topology/flowhs-storm-topology/src/main/java/org/openkilda/wfm/topology/flowhs/fsm/mirrorpoint/delete/actions/EmitCommandRequestsAction.java

```diff
@@ -75,54 +71,55 @@ protected void perform(State from, State to,
                         format("Flow mirror points for flow path %s and mirror switch id %s not found",
                                 flowPathId, mirrorSwitchId)));
 
-        FlowCommandBuilder commandBuilder = commandBuilderFactory.getBuilder(flow.getEncapsulationType());
-        Collection<FlowSegmentRequestFactory> commands =
-                buildCommands(commandBuilder, stateMachine, flow, mirrorPoints);
+        List<SpeakerData> installSpeakerCommands = new ArrayList<>();
```

Comment:
Not sure about it. Here we have 3 lists. To create a map of list we will need create keys for this lists. It can be enum keys (need to create new class) or just strings (need to create constants). 
It will require additional shared classes/variables. What we will get from using map of lists?

Reply:
I thought that maybe It could make the code shorter. But, yes, I see your point.